src/oss-find-squats-lib/MutateExtension.cs

```diff
@@ -165,7 +165,7 @@ public static async IAsyncEnumerable<FindPackageSquatResult> EnumerateExistingSq
                     FindPackageSquatResult? res = null;
                     try
                     {
-                        if (await manager.PackageExists(candidatePurl, false))
+                        if (await manager.PackageExists(candidatePurl, useCache: false))
```

Comment:
This shouldn't be hard coded false. Might need to introduce a parameter.

Reply:
Added to MutationOptions